### DISCUSSION TITLE
SIG Robotics takes place as a SIG working group?

### DISCUSSION THREAD

**Discussion by fujitatomoya** [kubeedge/kubeedge#4511] [General]
I was checking the following community activity called SIG Robotics for KubeEdge.

https://github.com/fujitatomoya/community/blob/master/sig-robotics/README.md

But almost everything about community information is empty,

 * [Meeting notes and Agenda]().
 * [Meeting Calendar]() | [Subscribe]()

just wondering this SIG takes place as community working group or not?

**Comment by fujitatomoya** [comment]
I would like to join this meeting if this takes place, this is really interesting for me as a member of [ROS Technical Steering Committee](https://docs.ros.org/en/rolling/The-ROS2-Project/Governance.html#technical-steering-committee-tsc)

CC: @fisherxu @crystaldust

**Reply by JoeyHwong-gk** [reply]
> I would like to join this meeting if this takes place, this is really interesting for me as a member of [ROS Technical Steering Committee](https://docs.ros.org/en/rolling/The-ROS2-Project/Governance.html#technical-steering-committee-tsc)
> 
> CC: @fisherxu @crystaldust


Dear  @fujitatomoya, We had a very pleasant online meeting in last May . As a member of sig robotics, I feel very honored that you can pay attention to our sig. Because the Chinese holiday is approaching, we plan to update our meeting schedule in the coming February, as well as a series of open source works on robotics based on kubeEdge. 
I will send you an invitation email. If you, or any-other followers, who are willing to participate in our roadmap formulation, I hope to invite you to participate in the discussion of community work in the community meeting in the near future. I hope you will not feel Abrupt, please pay more attention to our [community](https://github.com/kubeedge/community/tree/master/sig-robotics).

best!

**Reply by fujitatomoya** [reply]
@JoeyHwong-gk thanks for the info, I will be waiting for the invitation then. happy holidays!
